libs/web3/src/lib/use-vega-transaction-toasts.spec.tsx

```diff
@@ -59,20 +60,64 @@ jest.mock('@vegaprotocol/orders', () => {
     ...jest.requireActual('@vegaprotocol/orders'),
     useOrderByIdQuery: jest.fn(({ variables: { orderId } }) => {
       if (orderId === '0') {
+        const data: OrderByIdQuery = {
+          orderByID: {
+            id: '0',
+            side: 'SIDE_BUY',
+            size: '10',
+            remaining: '10',
+            timeInForce: 'TIME_IN_FORCE_FOK',
+            type: 'TYPE_MARKET',
+            price: '1234',
+            createdAt: new Date(),
+            status: 'STATUS_ACTIVE',
+            market: { id: 'market-1' },
+          },
+        } as OrderByIdQuery;
```

Comment:
Would be better to remove the type cast and use the correct enums

Reply:
The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables